STM32Cube/Tasks/millis.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * millis.c
+ *
+ *  Created on: May 22, 2024
+ *      Author: sengu
+ */
+
+// millis.c
+#include "millis.h"
+#include "stm32h7xx_hal.h"
+
+extern TIM_HandleTypeDef htim2;
+
+float millis(void) {
```

Comment:
slightly confused why this millis.c file is needed if it's just 1 helper function?

Reply:
its so that all the other threads can use the helper function, not just controller.